guides/common/modules/proc_upgrading-the-external-database.adoc

```diff
@@ -0,0 +1,23 @@
+[id="Upgrading_the_External_Database_{context}"]
+= Upgrading the External Database
+
+You can upgrade an external database from {EL} 7 to {EL} 8 while upgrading {Project} from {ProjectVersionPrevious} to {ProjectVersion}.
+
+.Prerequisites
+* Create a new {EL} 8 based host for PostgreSQL server that follows the external database on {EL} 8 documentation.
+For more information, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/deploying_different_types_of_servers/using-databases#migrating-to-a-rhel-8-version-of-postgresql_using-postgresql[Migrating to a RHEL 8 version of PostgreSQL].
+
+.Procedure
+. Create a backup.
+. If you keep the same hostname on the new PostgreSQL server, update the DNS record for the host to prevent the need to do updates on {Project} itself.
+. If the new PostgreSQL server has a new host name or a different CNAME, update the reference to the external database on {Project}:
```

Comment:
If you perform any of these steps *before* restoring the DB on the new system, you will see errors, which will confuse the reader (especially if you use the Installer method and the installer will fail).

Reply:
I'd move them towards the end, see also the wording suggestion in https://github.com/theforeman/foreman-documentation/pull/1614#discussion_r1021241684

Reply:
There really wasn't a place to move these since there weren't many steps in the Procedure to begin with but it has been reworded.